README.md

```diff
@@ -19,7 +19,7 @@ To learn more about Astria, please visit [astria.org](https://astria.org).
 
 * [conductor](https://github.com/astriaorg/astria/tree/main/crates/astria-conductor):
   conducts blocks from the data availability layer to the execution layer.
-* [proto](https://github.com/astriaorg/astria/tree/main/crates/astria-proto):
+* [proto](https://github.com/astriaorg/astria/tree/main/proto):
   relevant protobufs for Astria types.
```

Comment:
this should read something like `the protobuf spec to interact with Astria services`